aspnet/web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started.md

```diff
@@ -1,15 +1,15 @@
 ---
 uid: web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started
-title: "Getting Started | Microsoft Docs"
+title: Getting Started with ASP.NET | Microsoft Docs
 author: Rick-Anderson
-description: "WebMatrix is no longer recommended as an integrated development environment for ASP.NET Web Pages. Use Visual Studio or Visual Studio Code. This guidance an..."
+description: WebMatrix is no longer recommended as an integrated development environment for ASP.NET Web Pages. Use Visual Studio or Visual Studio Code. This guidance an...
```

Comment:
IF this doesn't resolve a warning, why are you making this change?

Reply:
This can be reverted to the original state.